docs/source/upgrade_notes.rst

```diff
@@ -117,6 +120,14 @@ Upgrade Notes
             type: "integer"
             required: true
 
+* ``version`` field in the pack metadata file must now contain a version string which is a
+  valid semver identifier - ``<major>.<minor>.<patch>``. For example ``0.1.0``, ``1.0.0,``2.0.1``,
+  etc.
+
+* Support for ``.gitinfo`` file has been removed and as such ``packs.info`` action has also been
+  removed. All the local pack directories are now direct git checkouts of the corresponding pack
+  repositories so this file is not needed anymore.
+
```

Comment:
Because `st2 run packs.install subtree=True` is not mentioned here,  I worry what else is missed? 

Suggest we consolidate all pack-related changes for good readibility, give a brief but complete description here, and leverage "transition to new pack" document, `/reference/exchange`.

3) StackStorm Exchange and st2contrib should be links.